src/test/java/org/folio/rest/impl/CodexInstanceResourceImplTest.java

```diff
@@ -47,7 +48,7 @@
 
   private final Header tenantHeader = new Header("X-Okapi-Tenant", "codexinstancesresourceimpltest");
   private final Header urlHeader = new Header("X-Okapi-Url", "https://localhost:" + okapiPort);
-  private final Header contentTypeHeader = new Header("Content-Type", "application/json");
+  private final Header contentTypeHeader = new Header("Content-IdentifierType", "application/json");
```

Comment:
probably a small typo for header `Content-Type`

Reply:
@natali-zaitseva 
thanks, corrected